scripts/irods/test/test_iadmin.py

```diff
@@ -1727,7 +1727,7 @@ def test_add_and_remove_specific_query(self):
         # Make specific query and run before it returns any results
         self.admin.assert_icommand(['iadmin', 'asq', specific_query, query_name])
         self.admin.assert_icommand(['iquest', '--sql', 'ls'], 'STDOUT', query_name)
-        self.admin.assert_icommand(['iquest', '--sql', query_name], 'STDOUT', 'No rows found')
+        self.admin.assert_icommand(['iquest', '--sql', query_name], 'STDOUT', 'CAT_NO_ROWS_FOUND: Nothing was found matching your query')
```

Comment:
Was this broken by #4745? Maybe we should link that in the commit message instead.

Reply:
yes, confirmed - i broke that.   please link accordingly.

Reply:
yes.

Reply:
done.